reuse/contribute-to-docs.txt

```diff
@@ -28,18 +28,101 @@ Use the :guilabel:`Give feedback` button at the top of any page to create a GitH
 New content
 -----------
 
-While contributing new content, it is easier to work with the docs on your local machine. You can submit a PR after all the checks have passed and things looks satisfactory. You'll need ``make`` and ``python3`` installed on your system.
+When adding new content, it's easier to work with the documentation on your local machine. Once you've made your changes, ensure all checks have passed and everything looks satisfactory before submitting a pull request (PR). To run the checks, you must have ``make`` and ``python3`` installed on your system.
```

Comment:
You need 'make' and 'python3' not just to run the checks, but also to locally build and see the results. So maybe we could remove the 'to run the checks' part and move it ahead a little. So something like: 
".. it's easier to work with the documentation on your local machine. For this, you'll need ``make`` and ``python3`` installed on your system."

Also, 'you'll need' sounds less prescriptive (& more friendly) than 'you must have', so maybe we could keep that as it is.